docs/src/test-reporters-js.md

```diff
@@ -516,3 +516,20 @@ allure generate ./allure-results --clean && allure open ./allure-report
 ```
 
 See [here](https://www.npmjs.com/package/allure-playwright) for more information.
+
+### Tesults reporter
```

Comment:
The reason allure is above was that it was a first third-party reporter. Now that we have established stable reporter API we can transform this section into the the list of the links to the reporters (without examples). I.e. feel free to remove the allure steps and leave two links to Allure and Tesults here.